Sources/CorePayments/Networking/APIClient.swift

```diff
@@ -51,12 +52,26 @@ public class APIClient {
         return try await http.performRequest(httpRequest)
     }
     
-    // TODO: - Add GraphQL equivalent request type & function
-    // public func fetch(request: GraphQLRequest) async throws -> HTTPResponse { }
+    /// :nodoc:
+    public func fetch(request: GraphQLRequest) async throws -> HTTPResponse {
+        let url = try constructGraphQLURL(queryName: request.queryNameForURL)
+
+        let postBody = GraphQLHTTPPostBody(query: request.query, variables: request.variables)
+        let postData = try JSONEncoder().encode(postBody)
+                
+        let httpRequest = HTTPRequest(
```

Comment:
If we single line this is it above our linter line length? May be nice to just group all of the properties together but not sure if this is more readable this way.

Reply:
It does not violate our line length, but I thought it was a little cleaner. Happy to go either way though!

Reply:
Either way is fine, was more just curious. If it makes more sense to have things grouped together a particular way that's fine too.

Reply:
Not sure what you mean by "group all of these properties together"?

Reply:
Ok I single-lined both the `HTTPRequest()` definitions (see commit 7417e153d6284d7ffb41c7a14487b812e743239f)

Reply:
> Not sure what you mean by "group all of these properties together"? You mean single line it?

Ah, sorry was referencing [this comment](https://github.com/paypal/paypal-ios/pull/182#discussion_r1296358473)

Reply:
That link takes me back to:

> If we single line this is it above our linter line length? May be nice to just group all of the properties together but not sure if this is more readable this way.

Do  you mean to remove the newlines between [these variables](https://github.com/paypal/paypal-ios/blob/ff3ed9b993134432f620d30c407e34e570f51e4c/Sources/CorePayments/Networking/APIClient.swift#L37-L45)?

Reply:
Weird, idk why GH is linking to that. This comment above is the one I was trying to reference:

> I did it intentionally to group postBody and postData together, but down to remove this newline if it's harder to read

Not a blocker though, whatever you feel is best